mesa_geo/raster_layers.py

```diff
@@ -218,15 +218,18 @@ class RasterLayer(RasterBase):
     _neighborhood_cache: dict[Any, list[Coordinate]]
     _attributes: set[str]
 
-    def __init__(self, width, height, crs, total_bounds, cell_cls: type[Cell] = Cell):
+    def __init__(
+        self, width, height, crs, total_bounds, model, cell_cls: type[Cell] = Cell
+    ):
         super().__init__(width, height, crs, total_bounds)
+        self.model = model
```

Comment:
For transparency, this line breaks one of our GIS example models, because it tries to initialize a `RasterLayer` without passing the model variable (which wasn't required before.

Which is not necessarily a bad thing, but it does indicate that Mesa-geo maybe doesn't yet has the stability and scrutiny that Mesa itself has. Which also isn't a bad thing, as long as we're 0.x it allows us to break stuff every minor release according to SemVer.

While in a 0.x stage these kind of monster PRs are acceptable, for a stable 1.0 library it would be very challenging.

It also proves how valuable it's to have your example models tested in CI.

Reply:
Aside from the RasterLayers, quite some other elements now required a model as input. Which isn’t a bad thing, but does break existing models, of which we should inform our users.

See for more details:
- https://github.com/projectmesa/mesa-examples/pull/177